### ISSUE TITLE
Genium support for MAC

### ISSUE THREAD

**Issue by gowthamjs** [Genium-Framework/Appium-Support#8]
Hi,

Thanks for the Genium-Framework. Its great to have such a framework to start appium server through java code. I was trying to use the same on my scripts. 

I tried to follow the steps mentioned in the Wiki page, when I execute the mvn test command, I was getting the below error.

Tests in error: 
  print(Run.AppTest): The file you provided was not found on your HDD. You provided the following file path: /Applications/Appium.app/Contents/Resources/node_modules/appium/bin/appium.js

I understand the appium.js doesn't exist anymore, so I was trying to use the following code to set the path to main.js

```
            AppiumServer _appiumServer = new AppiumServer(new File("/Applications/Appium.app"),new  File("/Applications/Appium.app/Contents/Resources/node_modules/appium/build/lib/main.js"),serverArguments);
```

Here is the output:

Sep 16, 2016 1:14:48 PM com.github.genium_framework.appium.support.server.AppiumServer startServer
INFO: Server has not started yet. Trying again in one second...
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 30.286 sec <<< FAILURE!
print(Run.AppTest)  Time elapsed: 30.242 sec  <<< ERROR!
com.github.genium_framework.server.exception.ServerTimeoutException: The server didn't start after 30000 milliseconds with error:

```
at com.github.genium_framework.appium.support.server.AppiumServer.startServer(AppiumServer.java:298)
at com.github.genium_framework.appium.support.server.AppiumServer.startServer(AppiumServer.java:245)
at Run.AppTest.print(AppTest.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
```

Results :

Tests in error: 
  print(Run.AppTest): The server didn't start after 30000 milliseconds with error:(..)

**Comment by Hassan-Radi** [comment]
Hi,

The problem here is that in the latest versions on Appium, they changed the structure a little bit and the files which where used to start the server are now located elsewhere with new names.

The issues you are having are because the library is trying to locate the node executable files in the Appium installation directory and failing since the location you provided is the main.js file location and not the directory to the Appium installation folder as you should have provided.

Long story short, to solve your problem just provide both locations:
- The main.js file location
- node executable file location

by doing [this](https://github.com/Genium-Framework/Appium-Support/wiki#note-2):

> AppiumServer _appiumServer = new AppiumServer(new File("C:/node_Executable_File_Path"), new File("C:/appium_JavaScript_File_Path"), serverArguments);

**Comment by gowthamjs** [comment]
Hi Hassan,

Thanks for the quick response.

I have resolved this issue with this statement,
        AppiumServer _appiumServer = new AppiumServer(new File("/Applications/Appium.app/Contents/Resources/node/bin/node"),new  File("/Applications/Appium.app/Contents/Resources/node_modules/appium/build/lib/main.js"),serverArguments);

Appreciate your quick help on this.
